tools/cldr-apps/pom.xml

```diff
@@ -0,0 +1,243 @@
+			<artifactId>mysql-connector-java</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.google.myanmartools</groupId>
+			<artifactId>myanmar-tools</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>xml-apis</groupId>
+			<artifactId>xml-apis</artifactId>
+		</dependency>
+	</dependencies>
+	<build>
+		<finalName>cldr-apps</finalName>
+		<sourceDirectory>src</sourceDirectory> <!-- TODO: HACK! until we can pivot src dirs -->
+		<pluginManagement><!-- lock down plugins versions to avoid using Maven 
+				defaults (may be moved to parent pom) -->
+			<plugins>
+				<plugin>
+					<artifactId>maven-clean-plugin</artifactId>
+					<version>3.1.0</version>
+				</plugin>
+				<!-- see http://maven.apache.org/ref/current/maven-core/default-bindings.html#Plugin_bindings_for_war_packaging -->
+				<plugin>
+					<artifactId>maven-resources-plugin</artifactId>
+					<version>3.0.2</version>
+				</plugin>
+				<plugin>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>3.8.0</version>
+					<configuration>
+						<includes>
+							<include>org/**/*.java</include>
+						</includes>
+					</configuration>
+				</plugin>
+				<plugin>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<version>${maven-surefire-plugin-version}</version>
+					<configuration>
+						<systemPropertyVariables>
+							<CLDR_DIR>${project.basedir}/../../</CLDR_DIR>
+							<CLDR_ENVIRONMENT>UNITTEST</CLDR_ENVIRONMENT>
+							<java.awt.headless>true</java.awt.headless>
+							<!-- <rununittest.arg>-n -q</rununittest.arg> -->
+						</systemPropertyVariables>
+					</configuration>
+				</plugin>
+				<plugin>
+					<artifactId>maven-war-plugin</artifactId>
+					<version>3.2.2</version>
+					<configuration>
+						<webResources>
+							<resource>
+								<!-- this is relative to the pom.xml directory -->
+								<directory>WebContent</directory>
+								<excludes>
+									<exclude>WEB-INF/lib/*</exclude>
+								</excludes>
+							</resource>
+						</webResources>
+						<archive>
+							<manifestEntries>
+								<Built-By>${user.name}</Built-By>
+								<Build-Time>${maven.build.timestamp}</Build-Time>
+								<CLDR-Apps-Git-Commit>${buildNumber}</CLDR-Apps-Git-Commit>
+								<CLDR-Apps-Git-Branch>${scmBranch}</CLDR-Apps-Git-Branch>
+							</manifestEntries>
+						</archive>
+					</configuration>
+
+				</plugin>
+				<plugin>
+					<artifactId>maven-install-plugin</artifactId>
+					<version>2.5.2</version>
+				</plugin>
+				<plugin>
+					<artifactId>maven-deploy-plugin</artifactId>
+					<version>2.8.2</version>
+				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>buildnumber-maven-plugin</artifactId>
+					<version>1.4</version>
+					<executions>
```

Comment:
I didn't remember / realize this, but pluginManagement and dependencyManagement propagate the configuration and other information for their respective plugin and dependency declarations, not just the version numbers.  These configuration details are duplicated in the root pom.xml, which `tools/java/pom.xml` is inheriting.  You could remove these details here, similarly, to clean up and allow inheritance.  Of course, verify that what I said is correct and the behavior is unchanged.

Or, alternatively, if you prefer, you could not use the inheritance mechanism for the configuration details, leaving it to be explicitly configured only in leaf pom.xml files and not in the root pom.xml.

I don't have a strong opinion on that, either way.

Reply:
@echeran
- I've moved *all* `*Management` into the parent pom
- the only common configuration that makes sense to make common (and I have) is the githash stuff, everything else is per-project.